STYLE.md

```diff
@@ -246,8 +246,50 @@ such as the current time. Instead, mocks such as
 [`MockRandomGenerator`](test/mocks/runtime/mocks.h) and
 [`Mock*TimeSource`](test/mocks/common.h) should be used.
 
+# Python coding style
+
+## pep8
+
+All python code should meet the pep8/pycodestyle standard.
+
+The exception to this is for positioning of boolean operators in multiline conjunctions (W503/W504)
+
+For example the following code is valid:
+
+```python
+
+some_aggregate_condition = (
+    a_condition
+    and (a_sub_condition
+         or another_sub_condition))
+
+```
+
+In this example the line split occurs before the `and` and `or` which are placed at the
```

Comment:
Will this be lintable? I'd like this not to be a subject of code reviews :) FWIW this is the kind of thing that clang-format does well at.